plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -2108,8 +2109,12 @@ public ConnectorAnalyzeMetadata getStatisticsCollectionMetadata(ConnectorSession
         MetadataEntry metadata = handle.getMetadataEntry();
 
         Optional<Instant> filesModifiedAfterFromProperties = getFilesModifiedAfterProperty(analyzeProperties);
+        Optional<Boolean> ignoreOldStats = getRecalculateAllStatisticsProperty(analyzeProperties);
```

Comment:
Let's name this boolean consistently throughout, rather than `recalculateAll` some and `ignoreOld` elsewhere. Personally I prefer affirmative names like `recalculateAll` or `forceFullRefresh`.